src/components/select/select.js

```diff
@@ -160,15 +189,17 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $interpolate) {
 
       function openSelect() {
         scope.$evalAsync(function() {
+          var selectEl = selectTemplate.clone();
+          selectEl.find('md-select-menu').data('$ngModelCtrl', ngModel);
```

Comment:
Just set `element.data('$ngModelController')` and the require will work as intended on line 216.